### PR TITLE
Cleaned up codebase for layouts rebased

### LINKED ISSUE #923: Cleanup codebase for layout components
@Nuzair46 _a

### PR DESCRIPTION
**Pull request by Nuzair46**
closes #923 
#927 rebased with master

### CODE REVIEW

**Comment by Nuzair46** [comment]
@amaldinesh7 _a, Please review. Thanks.

File: lib/components/layouts/Sidebar/Footer/FooterLink/index.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import { Link } from "./Link";
+import NavIconWrapper from "../../NavIconWrapper";
+import Typography from "../../../../Typography";
+
+export const FooterLink = ({ props, collapsed, tooltipStyle }) => {
```
Comment (amaldinesh7) [inline lib/components/layouts/Sidebar/Footer/FooterLink/index.js:6]: Make the export `default export`

File: lib/components/layouts/Sidebar/Footer/FooterLink/Link.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
+import React, { forwardRef } from "react";
+import classNames from "classnames";
+import { NavLink } from "react-router-dom";
+
+const LINK_CLASS_NAMES =
+  "flex items-center no-underline select-none neeto-ui-sidebar__link";
+
+export const Link = forwardRef(({ children, to, href, ...otherProps }, ref) => {
```
Comment (amaldinesh7) [inline lib/components/layouts/Sidebar/Footer/FooterLink/Link.js:8]: Make the export `default export`

File: lib/components/layouts/Sidebar/ProfileSection/constants.js [hunk 3]
```diff
@@ -0,0 +1,2 @@
+export const truncate = (str, length) =>
```
Comment (amaldinesh7) [inline lib/components/layouts/Sidebar/ProfileSection/constants.js:1]: This is not a constant. please rename the file to utils.

File: lib/components/layouts/MenuBar/index.js [hunk 4]
```diff
@@ -55,17 +59,22 @@ MenuBar.Search = Search;
...
   children: PropTypes.oneOfType([
     PropTypes.arrayOf(PropTypes.node),
     PropTypes.node,
   ]),
+  /**
+   * To specify whether the Menubar is visible or not.
+   */
   showMenu: PropTypes.bool,
+  /**
+   * To specify the class name for the Menubar.
```
Comment (amaldinesh7) [inline lib/components/layouts/MenuBar/index.js:75]:
```suggestion
   * To specify external classnames for the Menubar.
```

**Review by amaldinesh7 — CHANGES_REQUESTED** [review]
@Nuzair46 _A Please check the comments.

**Comment by Nuzair46** [comment]
@amaldinesh7 _a, Suggested changes are made. Please review. Thanks.

**Pull request merged**